app/services/analytics_events.rb

```diff
@@ -2302,6 +2302,7 @@ def logout_initiated(
   # @param [Hash] errors Authentication error reasons, if unsuccessful
   # @param [String] context
   # @param [String] multi_factor_auth_method
+  # @param [DateTime] multi_factor_auth_method_created_at time auth method was created
```

Comment:
For awareness: In trying to verify how we'd query this, I wasn't able to get CloudWatch to produce results for date string filtering. Worst case we might need to convert this to another format like a timestamp number, but hopefully I'm just doing something wrong. Have you been able to devise a query using this type of data?

Slack thread here: https://gsa-tts.slack.com/archives/C5E7EJWF7/p1688563099123109